mpas_analysis/shared/plot/plotting.py

```diff
@@ -27,6 +28,187 @@
 
 from ..constants import constants
 
+
+def nino34_spectra_plot(config, f, ninoSpectra, confidence95, confidence99,
```

Comment:
I'd consider having default `xLabel` and `yLabel` arguments for enhanced clarity.